RideShare/src/app/app.component.spec.ts

```diff
@@ -2,15 +2,16 @@ import { TestBed, async } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
```

Comment:
App Component should have some tests

Reply:
App Component is not doing anything by itself, aside from displaying the <app-router>. Should we still have tests for it?